bot.py

```diff
@@ -22,6 +24,11 @@ def __init__(self, config: dict, startup_extensions: List[str]):
                              reactions=True, guild_reactions=True, dm_reactions=True, typing=True, guild_typing=True,
                              dm_typing=True
                          ))
+        fileConfig('logging.conf')
+        self.logger = logging.getLogger(f'10man.{__name__}')
+        self.logger.debug(f'Version = {__version__}')
```

Comment:
Could just be on my machine but these logs are showing on console but aren't showing on the log files